tools/dev-get-emails.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Returns a csv of all steward emails for all datasets in the dev environment
+
+set -eu
+
+read -rp "Enter your Broad email: " BROAD_EMAIL
+read -rp "Enter your dev email: " DEV_EMAIL
+
+gcloud auth login "${DEV_EMAIL}" 2>/dev/null
+TOKEN=$(gcloud auth print-access-token)
+gcloud auth login "${BROAD_EMAIL}" 2>/dev/null
+
+dataset_list() {
+    curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/api/repository/v1/datasets?direction=desc&limit=10000&offset=0&sort=created_date" -H "accept: application/json" -H "authorization: Bearer ${TOKEN}" | jq -r '.items[] | "\(.id) \(.name)"'
+}
+
+dataset_policy() {
+    curl -s -X GET "https://sam.dsde-dev.broadinstitute.org/api/resources/v2/dataset/${1}/policies" -H "accept: application/json" -H "authorization: Bearer ${TOKEN}"
+}
+
+dataset_emails() {
+    IFS=$'\n'
+    ALL_EMAILS=()
+    for ROW in $(dataset_list) ; do
+        DATASET_ID=$(echo "${ROW}" | cut -d' ' -f1)
+        DATASET_NAME=$(echo "${ROW}" | cut -d' ' -f2)
+        DATASET_EMAILS=$(dataset_policy "${DATASET_ID}" | jq --arg NAME "${DATASET_NAME}" '.[] | select(.policyName == "steward") | .policy | {memberEmails} | . + {name: $NAME}')
```

Comment:
Is there a need to get emails for other policies as well?

Reply:
Not at the moment, but it's easy enough to replace the `policyName` key (or `select` both a `steward` and something else)

Reply:
You could accept the policy as an input with a default too
```
read -rp "Enter the policy [steward]: " POLICY
POLICY=${POLICY:-steward}
```